Tests/LinuxMain.swift

```diff
@@ -1,13 +1,10 @@
 #if os(Linux)
 
 import XCTest
-@testable import FluentPostgreSQLTests
+@testable import PostgreSQLDriverTests
```

Comment:
Perhaps we can open an issue to present the progress that porting PostgreSQLDriver to vapor2